src/wp-includes/taxonomy.php

```diff
@@ -3610,6 +3612,7 @@ function clean_term_cache( $ids, $taxonomy = '', $clean_taxonomy = true ) {
 function clean_taxonomy_cache( $taxonomy ) {
 	wp_cache_delete( 'all_ids', $taxonomy );
 	wp_cache_delete( 'get', $taxonomy );
+	wp_cache_delete( 'last_changed', 'terms' );
```

Comment:
I'm curious, how does calling this function "change" terms such that `last_changed` should be considered stale?

Also, why invalidate `last_changed` by deleting it rather than by updating it via `wp_cache_set_terms_last_changed()`?

Reply:
This is how caches are invalidated elsewhere

https://github.com/WordPress/wordpress-develop/blob/f09c56bfd3569fe7597dcce7e891bef495e04076/src/wp-includes/taxonomy.php#L2851
https://github.com/WordPress/wordpress-develop/blob/f09c56bfd3569fe7597dcce7e891bef495e04076/src/wp-includes/taxonomy.php#L2949